packages/account/src/Components/currency-selector/__tests__/currency-selector.spec.tsx

```diff
@@ -307,7 +319,9 @@ describe('<CurrencySelector/>', () => {
         expect(set_currency_btn).toBeInTheDocument();
         expect(set_currency_btn).toBeDisabled();
 
-        const tether = screen.getByRole('radio', { name: /tether erc20 \(eusdt\)/i });
+        const tether: HTMLInputElement = screen.getByRole('radio', {
+            name: /tether erc20 \(eusdt\)/i,
+        }) as HTMLInputElement;
```

Comment:
do we need here double types HTMLInputElement?

Reply:
I will modify this.